src/seedu/addressbook/data/person/Address.java

```diff
@@ -35,22 +43,28 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
+    
+    public String getAddress(){
+    	String address = new String();
+    	address = block.toString() + ", " + street.toString() + ", " + unit.toString() + ", " + street.toString() + ", " + postalCode.toString();
```

Comment:
This is a very long line. You can consider writing this in multiple lines. Consider declaring a variable for the delimiter.